### PR TITLE
Name some script command functions

### PR DESCRIPTION
**Pull request by cbt6**

### CODE REVIEW

File: src/unk_0203F6C4.c [hunk 1]
```diff
@@ -253,15 +253,15 @@ static BOOL ScrCmd_007(ScriptContext * param0);
 static BOOL ScrCmd_008(ScriptContext * param0);
 static BOOL ScrCmd_009(ScriptContext * param0);
 static BOOL ScrCmd_00A(ScriptContext * param0);
-static BOOL sub_0203F808(u16 param0, u16 param1);
+static BOOL Compare(u16 value0, u16 value1);
```
Comment (lhearachel) [inline src/unk_0203F6C4.c]: Can we move this declaration out of the block of command handler declarations? Caught me off-guard seeing it here, at first.

File: src/unk_0203F6C4.c [hunk 2]
```diff
@@ -1745,15 +1745,15 @@ static BOOL ScrCmd_00A (ScriptContext * param0)
...
         return 0;
-    } else if (param0 == param1) {
+    } else if (value0 == value1) {
         return 1;
+    } else {
+        return 2;
     }
-
-    return 2;
 }
```
Comment (lhearachel) [inline src/unk_0203F6C4.c]: So this *doesn't* return a `BOOL`, then? Should it be `enum ComparisonResult`?

File: src/unk_0203F6C4.c [hunk 3]
```diff
@@ -1817,34 +1817,25 @@ static BOOL ScrCmd_010 (ScriptContext * param0)
...
-
-    v0 = ScriptContext_GetVarPointer(param0);
-    v1 = *v0;
-    v2 = ScriptContext_ReadHalfWord(param0);
-
-    param0->comparisonResult = sub_0203F808(v1, v2);
-    return 0;
+    u16 value0 = *ScriptContext_GetVarPointer(ctx);
+    u16 value1 = ScriptContext_ReadHalfWord(ctx);
+    ctx->comparisonResult = Compare(value0, value1);
```
Comment (lhearachel) [inline src/unk_0203F6C4.c:1830]: Does this not match as:

```c
u16 *value0 = ScriptContext_GetVarPointer(ctx);
u16 value1 = ScriptContext_ReadHalfWord(ctx);
ctx->comparisonResult = Compare(*value0, value1);
```

File: src/unk_0203F6C4.c [hunk 4]
```diff
@@ -1817,34 +1817,25 @@ static BOOL ScrCmd_010 (ScriptContext * param0)
...
-    u16 * v0;
-    u16 * v1;
-
-    v0 = ScriptContext_GetVarPointer(param0);
-    v1 = ScriptContext_GetVarPointer(param0);
-    param0->comparisonResult = sub_0203F808(*v0, *v1);
-
-    return 0;
+    u16 * pointer0 = ScriptContext_GetVarPointer(ctx);
+    u16 * pointer1 = ScriptContext_GetVarPointer(ctx);
```
Comment (lhearachel) [inline src/unk_0203F6C4.c:1837]: I think these would be better named as `varN` to align with the function name.

File: src/unk_0203F6C4.c (hunk 3 above)
Comment (cbt6) [reply src/unk_0203F6C4.c:1830]: Does not match.

**Review by lhearachel — APPROVED** [review]

**Pull request merged**